web/app/js/components/HealthPane.jsx

```diff
@@ -39,7 +41,7 @@ export default class HealthPane extends React.Component {
   getHealthStats() {
     let inboundSr = this.getAvgSuccessRate(this.props.upstreamMetrics);
     let outboundSr = this.getAvgSuccessRate(this.props.downstreamMetrics);
-    let sr = _.isUndefined(this.props.currentSr) ? neutralSr : this.props.currentSr;
+    let sr = _.isUndefined(this.props.currentSr) ? null : this.props.currentSr;
```

Comment:
since you're checking _.isNil in `getHealthClassName ` there is no need for this `_isUndefined`